packages/gestalt-design-tokens/dist/ios/StyleDictionaryColor.m

```diff
@@ -0,0 +1,68 @@
+
+//
+// StyleDictionaryColor.m
+//
+
+// Do not edit directly
+// Generated on Tue, 10 Aug 2021 23:17:55 GMT
```

Comment:
@ayeshakmaz could we add a CI check to make sure these generated files are up to date on master?

Example for how we do it for CSS flow types: https://github.com/pinterest/gestalt/blob/c0c3fd53884c2b4e690f540b04870a4a219c73dc/scripts/test.sh#L29-L41